### ISSUE TITLE
MacOS Big Sur warning NSUserNotificationCenter

### ISSUE THREAD

**Issue by vivl4725** [fyne-io/fyne#1549]
### Describe the bug:
When go run on Mac OS Big Sur a minor warning pops up. 

### To Reproduce:
Steps to reproduce the behaviour:
1. On MacOS Big Sur, run App (go run).
2. Look in the terminal there is an error (warning) 

### Screenshots:
<img width="1108" alt="Снимок экрана 2020-11-15 в 16 23 24" src="https://user-images.githubusercontent.com/17576577/99186197-d393fe80-275f-11eb-9ba8-d5a4cf0ee2e5.png">

### Example code:

Any code, projects or examples

### Device:
 - OS: MacOS
 - Version: 11.0.1 Big Sur
 - Go version: 1.15.4 (darwin/amd64)
 - Fyne version: 1.4.0

**Comment by 103337373** [comment]
I have the same problem

**Comment by andydotxyz** [comment]
This will show for anyone compiling on BigSur I guess - thankfully not end-users running the apps.
It is a brand new deprecation and nothing to worry about. The message will go away once we stop using that API but there are probably 2 years in which before the API is removed.

Currently we have to balance supporting old macOS that Apple no longer supports, whilst trying to work on their cutting edge systems.

**Comment by ababo** [comment]
Really annoying...

**Comment by robb-coinbase** [comment]
Is there any way to simply turn off deprecation warnings for app devs who can't do anything about the actual issue?

**Comment by andydotxyz** [comment]
> Is there any way to simply turn off deprecation warnings for app devs who can't do anything about the actual issue?

We have not found a way to do that, I guess Apple really wants us to make sure these are fixed.
Of course the end users of your app on a macOS platform won't see the console messages :).
As with everything else we are working as fast as we can - any help is appreciated.

> @andydotxyz Is there a way to disable the output # fyne.io/fyne/app? Fyne is the only one of my app's libraries that sends output to stderr like this.

You'd need to be specific about the error messages, the line you included there is not output by us, so there must be something before/after that it relates to.

**Comment by robb-coinbase** [comment]
My workaround:

### scripts/quiet

```bash
#!/bin/bash
#
# Run the given command, discarding junk output from
# 3rd-party libraries.
#
# E.g.:
#    ./scripts/quiet go run main.go
#    ./scripts/quiet make test

$* 2> >(./scripts/stderr-filter >&2)
```

### scripts/stderr-filter

```bash
#!/bin/bash
#
# Filter out junk output from 3rd-party libraries.

egrep -v '(NSUser|fyne)'
```

**Comment by andydotxyz** [comment]
This is a good workaround thanks @robb-coinbase - however I would be cautious about ignoring Fyne logs, they should only appear when there is an error in an app or runtime.

**Comment by changkun** [comment]
We have two option for this:

1. silence all deprecation warnings (easy)
2. migrates all APIs (hard, and not backwards compatible) 

Which one is a preferred?

**Comment by andydotxyz** [comment]
I think this was fixed in 2.1
